core/src/main/kotlin/org/evomaster/core/output/naming/ActionTestCaseNamingStrategy.kt

```diff
@@ -14,6 +14,8 @@ import org.evomaster.core.sql.SqlAction
 abstract class ActionTestCaseNamingStrategy(
     solution: Solution<*>,
     private val languageConventionFormatter: LanguageConventionFormatter,
+    protected val maxTestCaseNameLength: Int,
+    private val testCasesSize: Int = solution.individuals.size,
```

Comment:
having it here a input parameter feels quite weird, considering it depends on `solution`. maybe use a field that is initialized in an `init{}` block?

Reply:
Yep, hadn't though of it, but will do so